tidy-data/01-Reshape-Data/01-Reshape-Data.Rmd

```diff
@@ -363,23 +371,34 @@ table4b %>% gather(key = "year", value = "population", -country)
 
 ### Converting output
 
-If you looked closely at your results in the previous exercises, you may have noticed something odd: the new year column contains character vectors. You can tell because R displays `<chr>` beneath the column name.
+If you looked closely at your results in the previous exercises, you may have noticed something odd: the new year column is a character vector. You can tell because R displays `<chr>` beneath the column name. `names_ptypes` and `values_ptypes` arguments take a list of column name-prototype (ptype) pairs defining the desired type of each newly created column.
```

Comment:
This syntax seems a bit hard compared to the previous way :sob:. I feel like we need to include that each item within the `list()` consists of the bare variable name from the *resulting* dataset on the left (so, we created `year` above in quotes, now it is not in quotes) and on the right the variable type (how do we know `integer()` is an option? I know they are in base, but not totally intuitive. And can we steer them to a help page with the other options?)

Reply:
I added a bit more detail..